specificatie/BRK-Bevragen/openapi.yaml

```diff
@@ -1054,8 +1057,8 @@ components:
           description: ""
         valuta:
           allOf:
-          - description: "Waardelijst wordt gepubliceerd op http://www.kadaster.nl/schemas/waardelijsten/Valuta"
           - $ref: "#/components/schemas/Waardelijst"
+          - description: "Waardelijst wordt gepubliceerd op http://www.kadaster.nl/schemas/waardelijsten/Valuta"
```

Comment:
Dit verschilt met de BAG. Daar hebben we ervoor gekozen dat we de allOf niet hiervoor te gebruiken. In OAS 3.1. is het wel toegestaan om een description als sibling aan een $ref toe te voegen

Reply:
Dat klopt, ik heb hiervoor gekozen om direct de link zichtbaar te maken. Ook in de grafische interface zodat developers daar direct op in kunnen klikken. Als dit een issue voor code-genereren oplevert doen we het anders. Maar dit levert ook geen warning op in 3.0.